cpp/powsybl-cpp/powsybl-api.h

```diff
@@ -378,13 +378,22 @@ typedef enum {
     THREE,
 } ThreeSide;
 
+typedef struct voltage_range_struct {
+    double minimum_nominal_voltage;
+    double maximum_nominal_voltage;
+    double voltage;
```

Comment:
Actually these parameters are not compulsory: either voltage or range coefficient should be given for each [mininum_nominal_voltage, maximum_nominal_voltage].

Reply:
in c/cpp double can be null ?